CONTRIBUTING.md

```diff
@@ -203,6 +203,40 @@ or
 $ yarn start --disable-host-check --public=$(echo ${server.dev-server} | sed -e s/https:\\/\\/// -e s/http:\\/\\/// -e s/\\///) --host="0.0.0.0" --env.target=${CHE_API_EXTERNAL%????}
 ```
 
+#### Step 7: Testing new workspace loader
+
+When workspace loader server is run, one probably need to test introduced changes.
+
+To safely test the changes it is better to create a separate workspace which will be used by new loader.
+But by default, Che doesn't allow to run more than one workspace simultaneously.
+To change this behaviour one need to set `che.limits.user.workspaces.run.count` Che property to value greater than `1`.
+In development environment that could be reached by adding `CHE_LIMITS_USER_WORKSPACES_RUN_COUNT` environment variable for Che server deployment.
```

Comment:
I think we should have separated docs section for CHE_LIMITS_USER_WORKSPACES_RUN_COUNT, because it is useful information not only for ws-loader development. But up to you, maybe my propose is out of the scope this pr.

Reply:
But seems we have some info from che-docs https://github.com/eclipse/che-docs/blob/569dec8754b86c28b9b93e82b1f16e0e516b37f4/src/main/pages/che-7/installation-guide/proc_configuring-namespace-strategies.adoc#one-namespace-per-user-strategy

Reply:
I agree that we have to have docs which cover most frequent settings/properties in details and explains how to set them for different platforms and different deployment types of Che server.
As you said, it is indeed out of scope of this section and probably it would be better to have a link to such docs, but I didn't find them. So, as for now, I think it is ok to keep simplified inline version (well, we can even omit this section, but I believe it is need for people who want to contribute, at least make the contribution process a bit easier).

Reply:
@AndrienkoAleksandr thanks for the link, but it explains namespace strategies, but not how to apply a Che property.